### PR TITLE
Fix RBD level coloring and filter

### PR DESCRIPTION
**Pull request by joverlee521**
## Description of proposed changes

Came down this rabbit hole when I was thinking about [augur export v2 and the filter footers](https://github.com/nextstrain/augur/pull/1384#pullrequestreview-1847695155). I noticed the [ncov build](https://next.nextstrain.org/ncov/gisaid/global/6m@2024-01-29) had empty filter footers for `level` and `rbd_level` and investigated. 

See commits for details.

## Testing

- [x] [Test build](https://github.com/nextstrain/ncov/actions/runs/7702226205) - visible at https://next.nextstrain.org/staging/ncov/gisaid/trial/fix-rbd/global/6m?c=rbd_level

## Release checklist

If this pull request introduces backward incompatible changes, complete the following steps for a new release of the workflow:

 - [ ] Determine the version number for the new release by incrementing [the most recent release](https://github.com/nextstrain/ncov/releases) (e.g., "v2" from "v1").
 - [ ] Update `docs/src/reference/change_log.md` in this pull request to document these changes and the new version number.
 - [ ] After merging, [create a new GitHub release](https://github.com/nextstrain/ncov/releases/new) with the new version number as the tag and release title.

If this pull request introduces new features, complete the following steps:

 - [x] Update `docs/src/reference/change_log.md` in this pull request to document these changes by the date they were added.

### CODE REVIEW

File: workflow/snakemake_rules/main_workflow.smk [hunk 1]
```diff
@@ -1304,11 +1304,13 @@ rule assign_rbd_levels:
         tree = "results/{build_name}/tree.nwk",
     params:
         config=config["files"]["rbd_level_definitions"],
-        basal_clade_label="21L (Omicron)"
+        basal_clade_label="21L (BA.2)"
```
Comment (victorlin) [inline workflow/snakemake_rules/main_workflow.smk:1307]: Typo in the message of d7bb79a56d30c20afb341c4ddd86421123036c71?

```diff
- implies that this shouldn't not be configurable
+ implies that this shouldn't be configurable
```

**Review by victorlin — APPROVED** [review]

File: workflow/snakemake_rules/main_workflow.smk (hunk 1 above)
Comment (joverlee521) [reply workflow/snakemake_rules/main_workflow.smk:1307]: Whoops! Thanks for catching that, fixed in 90a84cdd1e9f11fb4ecbe5184eb284a85d0ec29d

**Comment by joverlee521** [comment]
Waiting on [internal discussion](https://bedfordlab.slack.com/archives/CSKMU6YUC/p1706643360558109) of whether we should update the [RBD definitions](https://github.com/nextstrain/ncov/blob/7e4046a98ecfb4226b484eb6017847dc0cf09e08/defaults/rbd_levels.yaml) or just drop them entirely.

**Comment by joverlee521** [comment]
Closing in favor of https://github.com/nextstrain/ncov/pull/1097

**Comment by joverlee521** [comment]
Merging to get RBD-levels in a working state so we can easily reinstate the feature in the future.

**Pull request merged**